packages/eas-cli/package.json

```diff
@@ -21,16 +21,16 @@
     "@expo/package-manager": "1.1.2",
     "@expo/pkcs12": "0.0.8",
     "@expo/plist": "0.0.20",
-    "@expo/plugin-help": "5.1.22",
-    "@expo/plugin-warn-if-update-available": "2.5.0",
+    "@expo/plugin-help": "5.1.23",
+    "@expo/plugin-warn-if-update-available": "2.5.1",
     "@expo/prebuild-config": "6.2.5",
     "@expo/results": "1.0.0",
     "@expo/rudder-sdk-node": "1.1.1",
     "@expo/spawn-async": "1.7.0",
     "@expo/steps": "1.0.51",
     "@expo/timeago.js": "1.0.0",
-    "@oclif/core": "1.23.2",
-    "@oclif/plugin-autocomplete": "1.3.10",
+    "@oclif/core": "^1.26.2",
```

Comment:
Do we want to switch to `^` versions from strict versions?

Reply:
I don't see the harm in that, v1 is  no longer developed anyway afaik (again, current is v4)